xsd-parser-cli/src/main.rs

```diff
@@ -58,32 +60,33 @@ fn process_dir(input_path: &Path, output_path: &Path) -> Result<(), String> {
     Ok(())
 }
 
-fn process_single_file(input_path: &Path, output_path: Option<&str>) -> Result<(), String> {
+fn process_single_file(input_path: &Path, output_path: Option<&str>) -> anyhow::Result<()> {
     let text = load_file(input_path)?;
-    let rs_file = parse(text.as_str()).map_err(|_| "Error parsing file".to_string())?;
+    let rs_file = parse(text.as_str()).unwrap();
```

Comment:
This shouldn't be an `.unwrap()`. If `.context` doesn't work, use `.map_err` like before but creating an `anyhow::Error` with `anyhow::anyhow!`.